docs/Legal.md

```diff
@@ -1,355 +1,154 @@
-between you and the Company that arises in whole or in part from this Agreement or your use of
-the API or SDK shall be decided exclusively by a court of competent jurisdiction located in
-Santa Clara County, California, and you hereby consent to, and waive all defenses of lack of
-personal jurisdiction and forum non conveniens with respect to venue and jurisdiction in the state
-and federal courts of Santa Clara County, California.
-
-### 18\. NO ASSIGNMENT
-
-Licensee shall not assign, sublicense or delegate any of its rights or
-obligations under this Agreement to any third party without the prior written consent of Discord.
-Any assignment, sublicense or delegation in violation of this provision shall be null and void.
-
-### 19\. GOVERNMENT USE AND ACCESS
-
-The API and SDK are “commercial items” as
-that term is defined at 48 C.F.R. 2.101, consisting of “commercial computer software” and
-“commercial computer software documentation” as such terms are used in 48 C.F.R. 12.212.
-Any use, modification, derivative, reproduction, release, performance, display, disclosure or
-distribution of the API or SDK by any government entity is prohibited, except as expressly
-permitted by the terms of this Agreement. Additionally, any use by U.S. government entities
-must be in accordance with 48 C.F.R. 12.212 and 48 C.F.R. 227.7202-1 through 227.7202-4. The
-contractor/manufacturer is Discord Inc., 401 California Ave Burlingame, CA 94010. Licensee
-shall not allow or assist any government entities, law enforcement, or other organizations to
-conduct surveillance or obtain data using its access to the SDK or API in order to avoid serving
-legal process directly on Discord. Any such use by you for law enforcement purposes is a breach
-of this Agreement.
-
-### 20\. ACKNOWLEDGMENT
-
-Use of the API or SDK affirms that Licensee has read this
-Agreement, understands it, and agrees to be bound by its contents.
+## Effective date: August 15, 2020 
+
+## Posted date: July 1, 2020
+ 
+*For a link to the current terms, please see [here](https://github.com/discord/discord-api-docs/blob/b9edace323c9df64c79f104d85984690ae4e2977/docs/Legal.md).*
+
+Welcome to Discord’s APIs, software development kits (**“SDKs”**), and associated documentation (collectively, **"APIs"**). 
+
+Throughout this document, we use the word **_“Terms”_** to refer to the terms below, terms within the accompanying API and SDK documentation, our [Discord Developer Policy](#DOCS_POLICY/), and any applicable policies and guidelines. If there is a conflict between these terms and additional terms applicable to a given API, the additional terms will control for that conflict. 
+
+# Section 1: Acceptance of the Terms and Registration
+
+## a. Accepting the Terms
+
+By accessing or using our APIs you are agreeing to comply with the Terms. You are also agreeing that the Terms control your relationship with us. Please read all the Terms carefully. 
```

Comment:
I'm assuming by this you mean specifically for bots? Or does this apply to regular users accessing the API through the official client/game SDK?

Reply:
The term `APIs` includes the term `SDKs`.